misc/personal_info/personal_info.py

```diff
@@ -0,0 +1,135 @@
+import json
+import os
+import sys
+from typing import Set
+
+from talon import Context, Module, actions, fs, imgui, settings, ui
+
+mod = Module()
+mod.mode("personal_info")
+mod.setting(
+    "personal_info_auto_select",
+    type=int,
+    default=1,
+    desc="Auto select specified entry in json list, or none if 0",
+)
+mod.list("personal_info", desc="List of personal info populated by json file")
+ctx = Context()
+
+
+@mod.capture
+def personal_info(m) -> list:
+    """Returns a personal_info name"""
+
+
+@ctx.capture(rule="{user.personal_info}")
```

Comment:
fyi you don't need these captures if it's just a straight mapping

you can use the list in .talon like so
https://github.com/knausj85/knausj_talon/blob/master/lang/snippets.talon#L3

i learned this recently - can save some time:)

Reply:
oh, that might not work here, nvm:)